app/pulp/app/models/auth.py

```diff
@@ -0,0 +1,65 @@
+"""
+Django models related to the Authentication system
+"""
+from gettext import gettext as _
+
+from django.core import validators
+from django.contrib.auth.models import BaseUserManager, AbstractBaseUser, PermissionsMixin
+from django.db import models
+from django.utils import timezone
+
+
+class PulpUserManager(BaseUserManager):
+    """
+    A custom manager for PulpUser objects.
+    """
+    use_in_migrations = True
+
+    def _create_user(self, username, password, is_superuser):
+        """
+        Creates and saves a PulpUser with the given username and password.
+        """
+        now = timezone.now()
+        if not username:
+            raise ValueError('The given username must be set')
+        user = self.model(username=username, is_active=True, is_superuser=is_superuser,
+                          date_joined=now)
+        user.set_password(password)
+        user.save(using=self._db)
```

Comment:
@bmbouter I played around with the code and it looks like self._db is actually None. I am not sure that there is value is specifying it.  However, I am also hesitant to remove because Django's implementation of UserManager object uses it.